src/Core/Framework/Adapter/Cache/InvalidateCacheTask.php

```diff
@@ -2,10 +2,8 @@
 
 namespace Shopware\Core\Framework\Adapter\Cache;
 
-use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Core\Framework\MessageQueue\ScheduledTask\ScheduledTask;
-use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
 
 #[Package('framework')]
 class InvalidateCacheTask extends ScheduledTask
```

Comment:
should we set `shouldRescheduleOnFailure` to true now for this task? So even when it might fail (e.g. redis not available) it will be retried (with backoff interval) and would not be stopped completely after some retries?

Reply:
sounds reasonable